docs/response.md

```diff
@@ -28,12 +28,10 @@ namespace App;
 
 class MyResponseBuilder extends MarcinOrlowski\ResponseBuilder\ResponseBuilder
 {
-   protected static function buildResponse(bool $success, int $api_code, 
-                                           $message_or_api_code, array $lang_args = null,
-                                           $data = null, array $debug_data = null): array
+   protected function buildResponse(bool $success, int $api_code, $msg_or_api_code, array $placeholders = null, $data = null, array $debug_data = null): array
```

Comment:
Thanks for your fix. Can you please correct this line so it won't be longer than 120 chars per line (that was the reason of wrapping the original code)?

Reply:
@MarcinOrlowski fixed this issue as requested